lib/client/ingest/signal_fx_client.js

```diff
@@ -190,10 +192,12 @@ SignalFxClient.prototype.sendEvent = function (event) {
 
 
 SignalFxClient.prototype._retrieveAWSUniqueId = function (callback) {
+  var _this = this;
```

Comment:
while i see this approach used in other parts of the code, it shouldn't be necessary as a function tied to the prototype as the default "this" is correct.